Dockerfile

```
@@ -59,6 +59,7 @@ FROM linux AS spacemesh
 
 # Finally we copy the statically compiled Go binary.
 COPY --from=builder /src/build/go-spacemesh /bin/
+COPY --from=builder /src/build/service /bin/
```

How about calling it `smeshing-service`? A `service` alone is ambiguous (what is it? :))
```suggestion
COPY --from=builder /src/build/smeshing-service /bin/
```

---

This `service` binary is used to start 2 different modes. It has 2 commands, `node` and `activation` (later one will be renamed to smeshing or smesher - whatever you would prefer.

---

yeah let's definitely NOT name it `service` it will be nightmare to found in any process list or on windows it will be completely "weird'.
Why can't we name it go-spacemesh as previously?

---

@pigmej we can, but looks like you have ignored:
* there is already binary with this name, as the previously used binary is kept intact,
* there is a suggestion to start smeshing service from separate binary (so we need separate names for smeshing service and node service binaries).
So your suggestion just does not make sense while what suggested @poszu makes sense in the second case.